frontend/src/app/register/register.ctrl.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Controller for register page
+ */
+angular.module('flocs.user')
+.controller('registerCtrl', ['$scope','$location','$log','userDao', 
+    function($scope,$location,$log,userDao){
+
+	    function send(){
+            var username = $scope['username'];
+            $log.log($scope['username']);
+            var firstname = $scope['firstname'];
+            var lastname = $scope['lastname'];
+            var email = $scope['email'];
+			var passwd = $scope['password'];
+			if (userDao.registerUser(username,firstname,lastname,email,passwd) === true){
+				$location.url("/succes.html");
```

Comment:
We try to have all URLs used within the frontend application declared only once and at a single place - app/app.mdl.js This e.g. allows for easy change of any URL in the future and also one can see all possible states and corresponding URL without going through many files (+ little easier handling of parameters of the states/URLs).

You have already declared some states in app.mdl.js (namely 'register' and 'login' - although it's not clear to me, when the login state is used). You can navigate between states (= basically navigating between URLs) using $state.go('register') or $state.go('success') (and optionally given parameters object to them - of course, if there are any parameters of the state declared in app.mdl.js - as an example, see state 'task-preview-set' state in app.mdl.js). Details and examples: https://github.com/angular-ui/ui-router/wiki